packages/keyboard-direction-mixin/README.md

```diff
@@ -0,0 +1,9 @@
+# @vaadin/keyboard-direction-mixin
+
+This package provides `KeyboardDirectionMixin` that does the following:
+
+1. Add `keydown` listener on the element
+
+2. Navigate through `items` via arrow keys, <kbd>Home</knd> and <kbd>End</kbd>
```

Comment:
Typo in `Home</knd>` should be `Home</kbd>`

Reply:
Done.